src/test/java/calculator/ApplicationCustomTest.java

```diff
@@ -0,0 +1,99 @@
+package calculator;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.assertSimpleTest;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import org.junit.jupiter.api.Test;
+
+public class ApplicationCustomTest extends NsTest {
+
+    @Test
+    void 빈_문자열_입력() {
+        assertSimpleTest(() -> {
+            run("\n");
+            assertThat(output()).contains("결과 : 0");
+        });
+    }
+
+    @Test
+    void 숫자_하나_입력() {
+        assertSimpleTest(() -> {
+            run("1");
+            assertThat(output()).contains("결과 : 1");
+        });
+    }
+
+    @Test
+    void 쉼표_구분자() {
+        assertSimpleTest(() -> {
+            run("1,2,3");
+            assertThat(output()).contains("결과 : 6");
+        });
+    }
+
+    @Test
+    void 콜론_구분자() {
+        assertSimpleTest(() -> {
+            run("1:2:3");
+            assertThat(output()).contains("결과 : 6");
+        });
+    }
+
+    @Test
+    void 쉼표와_콜론_혼합_구분자() {
+        assertSimpleTest(() -> {
+            run("1,2:3");
+            assertThat(output()).contains("결과 : 6");
+        });
+    }
+
+    @Test
+    void 커스텀_구분자_사용() {
+        assertSimpleTest(() -> {
+            run("//;\\n1;2;3");
+            assertThat(output()).contains("결과 : 6");
+        });
+    }
+
```

Comment:
예외 입력에 대한 다양한 케이스를 고려한 부분이 인상깊었어요!
```suggestion
@Test
    void 커스텀_구분자_사용2() {
        assertSimpleTest(() -> {
            run("//;\\n1;2:3");
            assertThat(output()).contains("결과 : 6");
        });
    }
```
이처럼 커스텀 구분자와 기본 구분자가 혼용되었을 때에 대한 테스트 케이스도 추가해보면 어떨까요?